pkg/analyzer/analyzers/mailgun/mailgun.go

```diff
@@ -22,6 +19,15 @@ type Analyzer struct {
 	Cfg *config.Config
 }
 
+type SecretInfo struct {
```

Comment:
Now that we have Token Information and Domain Information, using `SecretInfo` like other analyzers makes more sense.